shared/reducers/wallets.js

```diff
@@ -75,31 +75,61 @@ export default function(state: Types.State = initialState, action: WalletsGen.Ac
     }
     case WalletsGen.setBuildingAmount:
       const {amount} = action.payload
-      return state.set('building', state.get('building').merge({amount}))
+      return state
```

Comment:
what's the difference here from `return state.get('builtPayment').merge(...)`?

Reply:
Ah, you're updating multiple things. Seems like there should be a way to do it with `state.merge(...)`